assignments/game-of-life/code.js

```diff
@@ -0,0 +1,54 @@
+drawFilledRect(0, 0, width, height, 'black')
+
+
+
+const makeArrayRow = () => {
+  let row = []
+  for (let i = 0; i < width; i += 5) {
+    if (Math.random() < .2) {
+      row.push(1)
+    } else {
+      row.push(0)
+    }
+  }
+  return row
+}
+const makeArray = () => {
+  let b = []
+  for (let i = 0; i < height / 5; i++) {
+    b.push(makeArrayRow())
+  }
+  return b;
+}
```

Comment:
So this is where we need to pass in the new argument to `makeArrayRow` but I'd also do a similar thing to this function and have it take the rows and columns as arguments:

```suggestion
const makeArray = (rows, columns) => {
  let b = []
  for (let i = 0; i < rows; i++) {
    b.push(makeArrayRow(columns))
  }
  return b;
}
```

In some sense this just passes the buck to whoever calls this function but it makes this function easier to understand and test.